index.html

```diff
@@ -508,19 +516,12 @@ <h3>
           <li>If this object's {{[[Released]]}} internal slot is `true`, return
           <a>a promise resolved with</a> `undefined`.
           </li>
-          <li>Otherwise, let |promise:Promise| be <a>a new promise</a>.
-          </li>
-          <li>Run the following steps <a>in parallel</a>:
-            <ol>
-              <li>Run <a>release a wake lock</a> with |lock:WakeLockSentinel|
-              set to this object and |type:WakeLockType| set to the value of
-              this object's {{WakeLockSentinel/type}} attribute.
-              </li>
-              <li>Resolve |promise|.
-              </li>
-            </ol>
+          <li>Otherwise, run <a>release a wake lock</a> with
+          |lock:WakeLockSentinel| set to this object and |type:WakeLockType|
+          set to the value of this object's {{WakeLockSentinel/type}}
```

Comment:
what object? "`this`"?

Reply:
Yes. The `WakeLockSentinel` object whose `release()` method is being invoked.